packages/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.0+1
+
+* Fixed a bug which the camera is not positioned correctly at map initialization(temporary workaround).
```

Comment:
nit: we can reference the github issue.
I would also mentioned that it's a work around, e.g "Fixed(worked around) a bug..."

Reply:
I did add the `workaround` in the end of the sentence. Did you mean to remove it and add `Fixed(worked around)`?

Reply:
ohh oops missed that, that looks good.

Reply:
still lets reference the github issue